openquakeplatform/openquakeplatform/vulnerability/urls.py

```diff
@@ -40,6 +40,7 @@
     url(r'^view/(?P<gen_info_id>\d+)(/(?P<action>[a-zA-Z_-]+)?)?$',
         views.show_general_information, name='view'),
     url(r'^data/(\d+)/$', views.general_information_json_get),
+    url(r'^data$', views.general_information_json_get_all),
```

Comment:
Consider to change `r'^data$'` into `r'^data/?$'` which will match also with the trailing slash. This should work with the right orders of the matches (see comment above)